core/src/main/java/org/jdbi/v3/core/mapper/reflect/ReflectionMappers.java

```diff
@@ -115,6 +118,35 @@ public ReflectionMappers setCaseChange(UnaryOperator<String> caseChange) {
         return this;
     }
 
+    /**
+     * Set the strategy Jdbi uses for Java accessibility rules.
+     * The legacy default is to call {@code setAccessible(true)} in certain cases when we try to use a Constructor, Method, or Field.
+     * In the future, this default will be changed to a no-op, to better interact with the Java module system.
+     */
+    @Beta
+    public ReflectionMappers setAccessibleObjectStrategy(Consumer<AccessibleObject> makeAccessible) {
+        this.makeAccessible = makeAccessible;
```

Comment:
this.accessibleStrategy = accessibleStrategy. Heh, why does that not complain about "shadowing a class field" ?